packages/back-end/src/routers/archetype/archetype.controller.ts

```diff
@@ -40,8 +40,6 @@ export const getArchetype = async (
 ) => {
   const { org, userId } = getContextFromReq(req);
 
-  req.checkPermissions("manageArchetype");
```

Comment:
I removed this here, and on line 73 below. For other globally scoped resources, we don't filter them out of view for a user even if the user has a global role of `noaccess`. 

In this scenario, if a user has a global `noaccess` role, and then project-specific roles that allow them to manage features, they wouldn't see any archetypes because we'd be filtering them all out. 

With this update, archetypes now behave similar to how resources like saved groups work.